ibc_data/main_contrasts.tsv

```diff
@@ -374,3 +385,23 @@ MDTB	semantic_hard-easy
 MDTB	2back_hard-easy
 MDTB	tom_belief-photo
 MDTB	search_hard-easy
+Emotion	negative-neutral
+Multimodal	animate-inanimate
+Multimodal	audio
```

Comment:
SHould we keep audio within main_contrast ? I'm not sure. Same thing for tactile and visual

Reply:
I added them because they are already sums of contrasts:
`audio` = ['audio_animal', 'audio_monkey', 'audio_nature', 'audio_silence', 'audio_speech', 'audio_tools', 'audio_voice']
etc...